### PR TITLE
Rebuilt the styleguide page

### PR DESCRIPTION
**Pull request by fragsalat**

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[ignore]
```
Comment (faboweb) [inline .flowconfig:1]: do we use flow?

**Review by faboweb — COMMENTED** [review]
What are the aurelia generators doing?

File: .flowconfig (hunk 1 above)
Comment (fragsalat) [reply .flowconfig:1]: I used it do get a better auto completion in intellij. We can kick this out if you want. The cool thing about flow is that it's optional so if you don't configure it in your ide it should not give errors

Comment (faboweb) [reply .flowconfig:1]: Good to know. I would leave it in then.

**Comment by fragsalat** [comment]
This project was scaffolded by aurelia cli. These provides pre configured ready to use project skeletons with an gulp build process. The generators are in place to generate new files with basic structure like gulp tasks or aurelia specifics like custom elements, custom attributes, binding behaviors and value converters. We actually don't need it but there was as well no reason to delete it.

**Comment by faboweb** [comment]
Ok. Just didn't understand what they are for. Let's merge then?

**Review by faboweb — APPROVED** [review]

**Pull request merged**